server/mergin/utils.py

```diff
@@ -1,31 +1,93 @@
 # Copyright (C) Lutra Consulting Limited
 #
 # SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-MerginMaps-Commercial
-from sqlalchemy import Column
 
+from collections import namedtuple
+from flask_sqlalchemy import Model
+from sqlalchemy import Column, JSON
+from sqlalchemy.sql.elements import UnaryExpression
+from typing import Optional
 
-def parse_order_params(cls, order_params):
-    """Parse order parameters in query string
+
+OrderParam = namedtuple("OrderParam", "name direction")
+
+
+def split_order_param(order_param: str) -> Optional[OrderParam]:
+    """Split db query order parameter"""
+    try:
+        col, order = order_param.strip().split(" ")
+    except ValueError:
+        return
+    if order.lower() in ["asc", "desc"]:
```

Comment:
Just small refactor, if possible:

Use an enum for order directions instead of string literals. Here, you can check for members of enum, not hardcoded list.